### DISCUSSION TITLE
Course content is visible in ALL tab

### DISCUSSION THREAD

**Discussion by komalm** [Sunbird-Ed/Community#355] [General Discussion]
Hello @keshavprasadms, @vinukumar-vs, @AmiableAnil, @rajeevsathish

**Site Environments Details** - SunbirdEd portal of version 4.4.1

**Steps To Reproduce** :- Click on **ALL** tab

**Expected Result** :- Content stiched to courses should not be there on the **ALL** tab
**Actual Result** :- Content which is stiched to course are also visible on **ALL** tab.

Please guide me here

cc @Renuka-S, @vinod303

**Comment by Krishnaj20** [comment]
@komalm - Content and courses published on SunbirdED have 'Public' visibility hence they are shown in the ALL tab.
